templates/base/eslintrc

```diff
@@ -129,6 +139,97 @@
         "max-len": [1, 100, 4],
         "no-bitwise": 2,
         "max-params": [1, 3],
-        "max-depth": [2, 2]
+        "max-depth": [2, 2],
+
+        /* no-unsafe-innerhtml rule */
+        "no-unsafe-innerhtml/no-unsafe-innerhtml" : 2,
+
+        /* scanJS rules */
```

Comment:
From here down it looks like the spacing is off on the rules. Looks like you are using tabs instead of spaces maybe? If you could convert them and get them indented the same as the comment and other rules that would be awesome